tabs/pid_tuning.js

```diff
@@ -8,9 +8,7 @@ var SUPEREXPO_FEATURE_BIT = 23;
 
 TABS.pid_tuning.initialize = function (callback) {
     var self = this;
-    if (GUI.active_tab != 'pid_tuning') {
-        GUI.active_tab = 'pid_tuning';
-    }
+    if (GUI.active_tab != 'pid_tuning') { GUI.active_tab = 'pid_tuning'; }
 
```

Comment:
Yeah, totally agree on this one, but it's quite subjective. I'm used to single line conditioned being inlined. Also in this particular case the condition can actually just go away.